draft-jasdips-regext-rdap-rpki.md

```diff
@@ -50,25 +50,31 @@ end, RPKI defines the following cryptographic profiles:
   autonomous system (AS, [@RFC4271]) for routing that CIDR address block.
 * Autonomous System Provider Authorization (ASPA, [@!I-D.ietf-sidrops-aspa-profile]) where an autonomous system number
   (ASN, [@!RFC5396]) holder cryptographically asserts about the provider AS for that ASN.
+* BGPSec Router Certificate ([@!RFC8209]) where an ASN holder cryptographically asserts that a router holding the
+  corresponding private key is authorized to emit secure route advertisements on behalf of the AS specified in the
+  certificate.
 
 This document defines a new RDAP extension, "rpki1", for accessing the RPKI registration data in the Regional Internet
-Registries (RIRs), including at national and local levels, for aforementioned RPKI profiles through RDAP. The intent is
-that such RDAP data can complement the existing RPKI diagnostic tools when troubleshooting a route hijack or leak, by
+Registries (RIRs), including at national and local levels, for aforementioned RPKI profiles through RDAP. The motivation
```

Comment:
Isn't the terminology for all RIRs, LIRs, NIRs supposed to be Internet Number Registry (INR)? If this is intended for NIRs and/or LIRs, then INR is the right term not RIR.

Reply:
> Isn't the terminology for all RIRs, LIRs, NIRs supposed to be Internet Number Registry (INR)? If this is intended for NIRs and/or LIRs, then INR is the right term not RIR.

Yes, will change.